test/pummel/test-crypto-dh.js

```diff
@@ -8,15 +8,30 @@ if (!common.hasCrypto) {
   return;
 }
 
-assert.throws(function() {
-  crypto.getDiffieHellman('unknown-group');
-});
-assert.throws(function() {
-  crypto.getDiffieHellman('modp1').setPrivateKey('');
-});
-assert.throws(function() {
-  crypto.getDiffieHellman('modp1').setPublicKey('');
-});
+assert.throws(
+  function() {
+    crypto.getDiffieHellman('unknown-group');
+  },
+  /^Unknown group$/,
```

Comment:
I don't think these regular expressions will work. I think you need to include the error type as well. In this case:

`/^Error: Unknown group$/`